File: Prova 1/2721_indecisaoRenas.c
#include <stdio.h>

int main() {
    int i, x, soma = 0;

    for (i = 0; i < 9; i++) {
        scanf("%d", &x);
        soma += x;
    }

    if (soma % 9 == 0)
        printf("Rudolph\n");
    else if (soma % 9 == 8)
        printf("Blitzen\n");
    else if (soma % 9 == 7)
        printf("Donner\n");
    else if (soma % 9 == 6)
        printf("Cupid\n");
    else if (soma % 9 == 5)
        printf("Comet\n");
    else if (soma % 9 == 4)
        printf("Vixen\n");
    else if (soma % 9 == 3)
        printf("Prancer\n");
    else if (soma % 9 == 2)
        printf("Dancer\n");
    else if (soma % 9 == 1)
        printf("Dasher\n");
    
    return 0;
}

File: Prova 2/2721_indecisaoRenas.py
n = input().split(' ') 

i = 0
soma = 0

for x in n: # faça algo para x com base n
    soma += int(x)

if (soma % 9 == 0):
	print("Rudolph")
elif (soma % 9 == 8):
        print("Blitzen")
elif (soma % 9 == 7):
        print("Donner")
elif (soma % 9 == 6):
        print("Cupid")
elif (soma % 9 == 5):
        print("Comet")
elif (soma % 9 == 4):
        print("Vixen")
elif (soma % 9 == 3):
        print("Prancer")
elif (soma % 9 == 2):
        print("Dancer")
elif (soma % 9 == 1):
        print("Dasher")